src/components/events-list/event-summary.js

```diff
@@ -1,44 +1,40 @@
+import { Table } from '@equinor/eds-core-react';
 import PropTypes from 'prop-types';
 import React from 'react';
 
-import RelativeToNow from '../time/relative-to-now';
-import WarningState from './warning-state';
 import EventType from './event-type';
-
+import WarningState from './warning-state';
+import RelativeToNow from '../time/relative-to-now';
 import eventModel from '../../models/event';
-import { isWarningEvent } from '../../utils/event-model';
-import { Table } from '@equinor/eds-core-react';
 
-const EventSummary = ({ event }) => {
-  return (
-    <>
-      <Table.Cell>
-        <RelativeToNow time={event.lastTimestamp} titlePrefix="Start" />
-      </Table.Cell>
-      <Table.Cell>
-        <EventType event={event}></EventType>
-      </Table.Cell>
-      <Table.Cell>
-        {event.involvedObjectKind}/{event.involvedObjectName}
-      </Table.Cell>
-      <Table.Cell className="wrap">
-        <span>
-          {event.reason} - {event.message}
-          {event.involvedObjectState &&
-            event.involvedObjectState.pod &&
-            event.involvedObjectState.pod.restartCount > 0 && (
-              <span>
-                . Restarted {event.involvedObjectState.pod.restartCount} times
-              </span>
-            )}
-        </span>
-      </Table.Cell>
-      <Table.Cell>
-        {isWarningEvent(event) && <WarningState event={event}></WarningState>}
-      </Table.Cell>
-    </>
-  );
-};
+const EventSummary = ({ event }) => (
+  <>
+    <Table.Cell>
+      <RelativeToNow
+        time={event.lastTimestamp}
+        titlePrefix="Start"
+        capitalize
+      />
+    </Table.Cell>
+    <Table.Cell>
+      <EventType event={event}></EventType>
+    </Table.Cell>
+    <Table.Cell>
+      {event.involvedObjectKind}/{event.involvedObjectName}
+    </Table.Cell>
+    <Table.Cell className="wrap">
+      {event.reason} - {event.message}
+      {event.involvedObjectState &&
+        event.involvedObjectState.pod &&
+        event.involvedObjectState.pod.restartCount > 0 && (
+          <>. Restarted {event.involvedObjectState.pod.restartCount} times</>
+        )}
+    </Table.Cell>
+    <Table.Cell>
+      <WarningState event={event}></WarningState>
```

Comment:
WarningState is only used here. Perhaps we should use StatusBadge directly? 
```suggestion
      {isEventObsolete(event) && (
        <StatusBadge type="warning">Obsolete</StatusBadge>
      )}
      {isEventResolved(event) && (
        <StatusBadge type="success">Resolved</StatusBadge>
      )}
```